specs/engine.spec/src/main/resources/META-INF/zilla/core.idl

```diff
@@ -100,4 +100,23 @@ scope core
             octets extension;
         }
     }
+
+    scope event
+    {
+        struct Event
+        {
+            int64 timestamp = 0;
+            int64 traceId = 0;
+        }
+
+        struct BindingEvent extends Event
+        {
+            int64 bindingId = 0;
+        }
+
+        struct CatalogEvent extends Event
+        {
+            int64 catalogId = 0;
+        }
```

Comment:
Perhaps  we can further simplify by having just `Event`, but addding `namespacedId`, which is a generalization of `bindingId`, `catalogId`, etc.